linker/Mono.Linker.Steps/BlacklistStep.cs

```diff
@@ -46,7 +46,7 @@ protected override bool ConditionToProcess()
 
 		protected override void Process ()
 		{
-			foreach (string name in Assembly.GetExecutingAssembly ().GetManifestResourceNames ()) {
+			foreach (string name in typeof (BlacklistStep).GetTypeInfo ().Assembly.GetManifestResourceNames ()) {
```

Comment:
See [1784](https://github.com/dotnet/corefx/issues/1784). It looks like some of the removed APIs are not available to build against, despite existing in the coreclr sources.


> It's quite strange that Assembly.Load(byte[]) exists in CoreCLR project: https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/Reflection/Assembly.cs#L392
> Can someone explain me why it's there and we can't use it?

> @exyi It actually gets modified to be internal in the build process. It is complicated by various factors around how this code gets shared in many different configurations.